src/viz-population-over-time/VizPopulationOverTimeContainer.js

```diff
@@ -0,0 +1,96 @@
+import { ascending } from "d3-array";
+import { parseISO, startOfMonth, sub } from "date-fns";
+import PropTypes from "prop-types";
+import React, { useMemo } from "react";
+import { THEME } from "../theme";
+import { addEmptyMonthsToData, recordIsTotalByDimension } from "../utils";
+import {
+  DEMOGRAPHIC_UNKNOWN,
+  DIMENSION_MAPPINGS,
+  DIMENSION_DATA_KEYS,
+  TOTAL_KEY,
+} from "../constants";
+import VizPopulationOverTime from "./VizPopulationOverTime";
+
+const EXPECTED_MONTHS = 240; // 20 years
```

Comment:
Would it be overkill to derive the total number of months from the API response itself, just in case there was a desire to change that number in the future without frontend work? I could see that being overkill.

Reply:
There is a minor edge case in that approach, wherein missing rows at the extreme edges of the reporting window would not get converted to zeroes; if we continue to assume that the present month is always included then that would only affect the earliest extremes. I traded that off for a consistent number of months by hard-coding it, but I wouldn't object to it going the other way if you think that would be preferable

Reply:
No, I buy this based on the edge case you pointed out.